src/Raven.Client/Documents/Operations/Operation.cs

```diff
@@ -37,7 +37,7 @@ public class Operation : IObserver<OperationStatusChange>
         private readonly RequestExecutor _requestExecutor;
         private readonly Func<IDatabaseChanges> _changes;
         private readonly DocumentConventions _conventions;
-        private readonly Task _additionalTask;
+        private Task _additionalTask;
```

Comment:
if possible, please give a meaningful name, hard to tell what is this `_additionalTask`